docs/patch.md

```diff
@@ -211,13 +211,15 @@ Current Value     | Given Value     | Result
 `1`               | `[2]`           | `[1, 2]`
 `[]`              | `[1]`           | `[1]`
 `[1]`             | `[2]`           | `[1, 2]`
+`['a', 'b']`      | `['b', 'c']`    | `['a', 'b', 'b', 'c']`
+`{a: 1}`          | `{a: 2}`        | `{a: 2}`
```

Comment:
Could you keep the object examples close together pls?

Reply:
No problem. Consider it done.